### PR TITLE
Enhancements to redesigned home page

### PR DESCRIPTION
**Pull request by francoislehoux-okta**
## Description:
- **What's changed?** Made the home page a component and reuses the default app layout + addressed a couple of minor issues

### Resolves:
* [OKTA-359135](https://oktainc.atlassian.net/browse/OKTA-359135)

### Screenshots:
Screen Shot 2021-01-12 at 4 00 43 PM
Screen Shot 2021-01-12 at 4 01 00 PM
...
Screen Shot 2021-01-12 at 4 01 07 PM

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/index.js [hunk 1]
```diff
@@ -14,9 +14,10 @@ module.exports = (options, ctx) => {
       $page.fullHeaders = resolveHeaders($page);
 
       // Set to true to enable the redesign in development
-      const redesign = process.env.REDESIGN || false;
+      // const redesign = process.env.REDESIGN || false;
       // Respect the redesign flag unless this is a production build
-      $page.redesign = !ctx.isProd && redesign;
+      // $page.redesign = !ctx.isProd && redesign;
+      $page.redesign = true;
```
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-theme-prose/index.js:20]: Note

File: packages/@okta/vuepress-theme-prose/layouts/Layout.vue [hunk 2]
```diff
@@ -4,44 +4,54 @@
       <HeaderRedesign v-if="$page.redesign" />
```
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-theme-prose/layouts/Layout.vue:4]: Formatted this file

File: packages/@okta/vuepress-site/.vuepress/config.js [hunk 3]
```diff
@@ -231,9 +231,9 @@ module.exports = {
         contact: {
           heading: 'Contact & Legal',
           items: [
-            { text: 'Contact our team', link: 'https://developer.okta.com/contact/' },
-            { text: 'Contact sales', link: 'https://developer.okta.com/contact-sales-enterprise/' },
-            { text: 'Terms & conditions', link: 'https://developer.okta.com/terms/' },
+            { text: 'Contact our team', link: 'https://www.okta.com/contact/' },
+            { text: 'Contact sales', link: 'https://www.okta.com/contact-sales/' },
+            { text: 'Terms & conditions', link: '/terms/' },
             { text: 'Privacy policy', link: 'https://developer.okta.com/privacy/' },
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-site/.vuepress/config.js:237]: This link actually redirects to https://www.okta.com/privacy-policy/ -- should we also change this?

File: packages/@okta/vuepress-theme-prose/components/Home.redesign.vue [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+                >
+                  Authenticate up to 1,000 users per month for free forever
+                </p>
+              </div>
+            </div>
+            <div class="row redesign-homepage--selector-block-margin ">
+              <div
+                class="col-xl-3 col-lg-6 col-md-6"
+                v-for="(tile, index) in $page.frontmatter.tiles"
+                :key="index"
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/components/Home.redesign.vue:43]: I've been curious if there is a functional difference between `v-bind:key` and `:key`?

File: packages/@okta/vuepress-theme-prose/global-components/SignUp.vue [hunk 5]
```diff
@@ -158,9 +158,9 @@
         <div class="consent--section" v-show="displayConsent">
           <p class="consent--section-text">
             By clicking “SIGN UP” I agree to the applicable Free Trial terms in
-            <a href="/terms">Okta’s Terms of Service</a> during my use of the
+            <a href="/terms" target="_blank">Okta’s Terms of Service</a> during my use of the
             Free Trial Service and Okta’s
-            <a href="https://www.okta.com/privacy-policy">Privacy Policy</a>. I further agree that
+            <a href="https://www.okta.com/privacy-policy" target="_blank">Privacy Policy</a>. I further agree that
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/global-components/SignUp.vue:163]: Should these also have `rel="noopener noreferrer"`?

File: packages/@okta/vuepress-theme-prose/components/Home.redesign.vue (hunk 4 above)
Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/components/Home.redesign.vue:43]: Comes down to preference I think: https://stackoverflow.com/a/55105206

Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/components/Home.redesign.vue:43]: Looks like `:` is the shorthand for `v-`

File: packages/@okta/vuepress-theme-prose/global-components/SignUp.vue (hunk 5 above)
Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/global-components/SignUp.vue:163]: I'll create a ticket to go over all of our new links and validate that this is the case, I know we have a couple of these missing.

Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/global-components/SignUp.vue:163]: https://oktainc.atlassian.net/browse/OKTA-360351?atlOrigin=eyJpIjoiOWNhZTBiMDI1MWNiNDQ1OTgyZTE0YzgwNTYxNzBjOGEiLCJwIjoiaiJ9

**Review by shawyu-okta — APPROVED** [review]

**Pull request merged**